projects/soft_999_Diana_Brebeanu_fw_102/src/main.c

```diff
@@ -0,0 +1,62 @@
+#include <stdint.h>  // for integer types
+
+#include "interrupt.h"  // interrupts are required for soft timers
+#include "log.h"
+#include "soft_timer.h"  // Software timers for scheduling
+#include "status.h"
+#include "test_helpers.h"
+#include "wait.h"  // for wait function
+
+// this test uses a single soft timer to increment counter_a every half a second, and counter_b
+// every second.
+
+typedef struct Counters {
+  uint16_t counter;
```

Comment:
The struct should consists of both counter_a and counter_b. It should also be of type uint8_t